widget/embedded/src/hooks/useSyncUrlAndStore/useSyncUrlAndStore.ts

```diff
@@ -73,6 +76,7 @@ export function useSyncUrlAndStore() {
         [SearchParams.TO_TOKEN]: tokenToSearchParam(toToken),
         [SearchParams.FROM_AMOUNT]: inputAmount,
         [SearchParams.AUTO_CONNECT]: autoConnect ?? undefined,
+        [SearchParams.CLIENT_URL]: clientUrl ?? undefined,
```

Comment:
Do we always need to include the clientUrl in the URL?

Reply:
When user activate `dynamicHeight` it needs to setup a communication channel using message passing which has need origin.
Do I'm missing any point here?